src/Modules/Drawer.react.js

```diff
@@ -129,6 +131,23 @@ class DrawerHeader extends Component {
             </header>
         );
     }
+
+    componentDidMount() {
+        this._onHeaderUpdate();
+    }
+
+    _onHeaderUpdate(node) {
+        const headerEl = ReactDOM.findDOMNode(this);
+
+        if (headerEl) {
+            const paddingTop = headerEl.offsetHeight;
+
+            if (paddingTop !== this._drawerContainerInnerPaddingTop) {
+                this._drawerContainerInnerPaddingTop = `${paddingTop + 27}px`;
```

Comment:
Is it maybe worth defining a constant for 27 as the default padding?  In case next week design team takes a fit and wants to update it to 33 or some other value; that way you only have to update it in one place not two places (here and up on line 109).  Just a thought.

Reply:
Good idea. Might as well implement it.